package.json

```diff
@@ -8,16 +8,19 @@
     "bootstrap": "lerna bootstrap",
     "build": "lerna run build",
     "lint": "eslint './**/**/*.ts'",
-    "lint-fix": "eslint './**/**/*.ts' --fix"
+    "lint-fix": "eslint './**/**/*.ts' --fix",
+    "docs": "typedoc --options typedoc.json"
   },
   "author": "",
   "devDependencies": {
+    "@types/jest": "^29.5.6",
```

Comment:
I wonder if this is why builds fail for me? Something related to the jest types is failing...

```
~/code/pse/maci on  feat/typedocs [⇕$!] via ⬢ v18.18.2
➜ npm run build

> maci@1.1.1 build
> lerna run build

lerna notice cli v6.0.3

    ✔  maci-crypto:build (2s)
    ✔  maci-domainobjs:build (2s)
    ✔  maci-core:build (2s)

    ✖  maci-circuits:build
       > maci-circuits@1.1.2 build
       > tsc

       ../node_modules/@types/jest/index.d.ts(2,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'beforeEach' must be of type 'HookFunction', but here has type 'Lifecycle'.
       ../node_modules/@types/jest/index.d.ts(4,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'afterEach' must be of type 'HookFunction', but here has type 'Lifecycle'.
       ../node_modules/@types/jest/index.d.ts(5,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'describe' must be of type 'SuiteFunction', but here has type 'Describe'.
       ../node_modules/@types/jest/index.d.ts(7,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'xdescribe' must be of type 'PendingSuiteFunction', but here has type 'Describe'.
       ../node_modules/@types/jest/index.d.ts(8,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'it' must be of type 'TestFunction', but here has type 'It'.
       ../node_modules/@types/jest/index.d.ts(10,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'xit' must be of type 'PendingTestFunction', but here has type 'It'.
       ../node_modules/@types/jest/index.d.ts(11,13): error TS2403: Subsequent variable declarations must have the same type.  Variable 'test' must be of type 'TestFunction', but here has type 'It'.
       ../node_modules/@types/jest/index.d.ts(1198,22): error TS2344: Type 'T' does not satisfy the constraint 'new (...args: any) => any'.
         Type 'abstract new (...args: any) => any' is not assignable to type 'new (...args: any) => any'.
           Cannot assign an abstract constructor type to a non-abstract constructor type.
       ../node_modules/@types/jest/index.d.ts(1199,31): error TS2344: Type 'T' does not satisfy the constraint 'new (...args: any) => any'.
         Type 'abstract new (...args: any) => any' is not assignable to type 'new (...args: any) => any'.
           Cannot assign an abstract constructor type to a non-abstract constructor type.


 ————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————

 >  Lerna (powered by Nx)   Ran target build for 7 projects (7s)

    ✔    3/4 succeeded [0 read from cache]

    ✖    1/4 targets failed, including the following:
         - maci-circuits:build
```